wsgi_kerberos.py

```diff
@@ -6,7 +6,6 @@
 import errno
 import kerberos
 import logging
-import os
```

Comment:
(noticed this was unused while I was working on this, so I removed it while I was in here)